web/less/room.less

```diff
@@ -73,7 +73,8 @@ body {
     float: left;
     flex-grow: 1;
     height: 100%;
-    padding: 0 30px;
+    padding: 0 30px 0 55px;
+    position: relative;
 
```

Comment:
With the addition of the new lock icon to the left of the room name, the room name is wrapping to more than two lines on narrow windows. (We want the room name to be no more than two lines.) And on mobile, the lock icon floats over the room name. I think changing `min-width` for `.room-info` to `160px` fixes the first issue. I'm not sure how we are going to accommodate the lock icon and a long room name on mobile.

Reply:
I added a commit to fix the long room name issue.